src/main/resources/maps/LoseLevel/10x12_LoseLevel.crf

```diff
@@ -0,0 +1,19 @@
+# Level that is shown when the player lost the game.
```

Comment:
This file will have to be redone when the new room loading has been implemented.